### ISSUE TITLE
Focus styles missing on "More..." link (top nav)

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#10512]
### Describe the problem and steps to reproduce it:

1. Use keyboard navigation on (with several <kbd>tab</kbd> to reach the top-nav): https://addons-dev.allizom.org/en-US/firefox/

### What happened?

No style:

![Screenshot 2021-04-29 at 14 28 07](https://user-images.githubusercontent.com/217628/116550485-1f45f900-a8f7-11eb-8301-2e41aa4cd9db.png)

### What did you expect to happen?

Some focus styles. FWIW, this is how the link right _before_ looks:

![Screenshot 2021-04-29 at 14 28 24](https://user-images.githubusercontent.com/217628/116550508-2967f780-a8f7-11eb-9072-81b825949deb.png)

**Comment by ioanarusiczki** [comment]
@willdurand Verified on AMO stage - FF88(Win10). 

![focus on More](https://user-images.githubusercontent.com/33448286/117285808-18783280-ae71-11eb-836d-c000649d4980.gif)
